public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -122,157 +110,263 @@ figure.image-display
+  The first file you should check out is `README.md`.
+  It has some basic information on how to use CLI commands.
+  Whenever you want to know more about how Angular-CLI works make sure to visit 
+  [the Angular-CLI repository](https://github.com/angular/angular-cli) and 
+  [Wiki](https://github.com/angular/angular-cli/wiki).
 
-:marked
-  ### The *main.ts* file
-  
-  How does Angular know what to do with our component?  We have to tell it. 
-  We _bootstrap_ our application in the file `main.ts`.
-  
-+makeExcerpt('src/main.ts', 'important')
+  Some of the generated files might be unfamiliar to you. 
+  Let's go over the goodie bag:
 
-:marked
-  We import the two things we need to launch the application:
-
-  1. Angular's browser `bootstrap` function
-  1. The application root component, `CliQuickstartAppComponent`.
-
-  Then we call `bootstrap` with `CliQuickstartAppComponent`.
-
-  ### Bootstrapping is platform-specific
-  Notice that we import the `bootstrap` function from `@angular/platform-browser-dynamic`,
-  not `@angular/core`.
-  Bootstrapping isn't core because there isn't a single way to bootstrap the app.
-  True, most applications that run in a browser call the bootstrap function from
-  this library.
-
-  But it is possible to load a component in a different environment.
-  We might load it on a mobile device with [Apache Cordova](https://cordova.apache.org/) or [NativeScript](https://www.nativescript.org/).
-  We might wish to render the first page of our application on the server
-  to improve launch performance or facilitate
-  [SEO](http://www.google.com/webmasters/docs/search-engine-optimization-starter-guide.pdf).
-  These targets require a different kind of bootstrap function that we'd import from a different library.
-
-  ### Why create a *main.ts* file?
-
-  Both `main.ts` and the application component files are tiny.
-  This is just a QuickStart.
-  We could have merged these two files into one
-  and spared ourselves some complexity.
-
-  We'd rather demonstrate the proper way to structure an Angular application.
-  App bootstrapping is a separate concern from presenting a view.
-  Mixing concerns creates difficulties down the road.
-  We might launch the `CliQuickstartAppComponent` in multiple environments with different bootstrappers.
-  Testing the component is much easier if it doesn't also try to run the entire application.
-  Let's make the small extra effort to do it *the right way*.
-  
-  ### Loading the application with SystemJS
-  
-  The CLI uses `System.js` to load the application. We just need to call `System.import` and pass it our `main.ts`
-  file to boot our application.
-  
-+makeExcerpt('src/index.html', 'import')
+block src-files
+  :marked
+    ### The `src` folder
+    Your app lives in the `src` folder.
+    All Angular components, templates, styles, images and anything else your app needs goes in here.
```

Comment:
"go here." instead of "goes in here."  Maybe this is better:

>Files that belong to the application go here, including components, templates, services, classes, styles, and images. Files that help you build and deploy the app belong outside of the `src/` folder.